tmt/steps/execute/internal.py

```diff
@@ -246,19 +214,75 @@ def _test_output_logger(
                 level=level,
                 topic=topic)
 
-        # Execute the test, save the output and return code
-        starttime = datetime.datetime.now(datetime.timezone.utc)
-        try:
-            output = guest.execute(
-                remote_command,
+        # This variable tracks execution time
+        # for test together with setup and cleanup scripts
+        time_spent = 0
+
+        def run_cmd(command: ShellScript, wrapper_filepath: Path) -> tmt.utils.CommandOutput:
+            nonlocal time_spent
+            friendly_command = str(command)
+            # Prepare the wrapper, push to guest
+            if test.framework == "shell":
+                command = ShellScript(f"{tmt.utils.SHELL_OPTIONS}; {command}")
+            self.debug('Test script', str(command), level=3)
+            self.write(wrapper_filepath, str(command), 'w')
```

Comment:
The wrapper path is no longer logged, please, revert that dropped line:

```python
logger.debug('test wrapper', str(test_wrapper_filepath))
```

Reply:
Fixed